src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.c

```diff
@@ -46,80 +77,1172 @@ const ActorInit En_Kakasi_InitVars = {
     (ActorFunc)EnKakasi_Draw,
 };
 
-#endif
+Vec3f D_80971DCC[] = {
+    { 0.0f, 60.0f, 60.0f },   { 40.0f, 40.0f, 50.0f },   { -40.0f, 40.0f, 50.0f },
+    { 40.0f, 20.0f, 110.0f }, { -40.0f, 20.0f, 110.0f }, { 0.0f, 80.0f, 60.0f },
+    { 50.0f, 40.0f, -30.0f }, { -50.0f, 40.0f, -30.0f }, { 0.0f, 50.0f, 60.0f },
+};
+Vec3f D_80971E38[] = {
+    {0.0f, 55.0f, 60.0f,},
+    {0.0f, 55.0f, 60.0f,},
+    {-80.0f, 35.0f, 15.0f,},
```

Comment:
```suggestion
    { 0.0f, 55.0f, 60.0f },
    { 0.0f, 55.0f, 60.0f },
    { -80.0f, 35.0f, 15.0f },
```
Make sure all these `Vec3f` have the proper spacing. Is that caught by formatter? I have no idea. I also don't think you need the internal trailing comma